sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/_tests_/sfas-integration.scheduler.e2e-spec.ts

```diff
@@ -156,6 +159,95 @@ describe(describeProcessorRootTest(QueueNames.SFASIntegration), () => {
     },
   );
 
+  it(
+    "Should add a SFAS part time application record when importing data from SFAS " +
+      "where the record type is the part time application data record.",
+    async () => {
+      // Arrange
+      // Queued job.
+      const job = createMock<Job<void>>();
+      mockDownloadFiles(sftpClientMock, [SFAS_SAIL_DATA_FILENAME]);
+      // Act
+      const processingResults = await processor.processSFASIntegrationFiles(
+        job,
+      );
+      // Assert
+      // Expect the file was deleted from SFTP.
+      expect(sftpClientMock.delete).toHaveBeenCalled();
+      // Expect the file contains 2 records.
+      expect(processingResults[0].summary[1]).toBe("File contains 2 records.");
+      const startDate = dayjs("20230801", "YYYYMMDD", false).format(
```

Comment:
I would say we can just use one query based on individual id to perform all the validations.

```ts
const sfasPartTimeApplications = await db.sfasPartTimeApplications.find({
        select: {
          startDate: true,
          endDate: true,
          CSGPAward: true,
          SBSDAward: true,
          CSPTAward: true,
          CSGDAward: true,
          BCAGAward: true,
          CSLPAward: true,
          programYearId: true,
        },
        where: {
          individualId: In([950000360, 950000361]),
        },
      });
      expect(sfasPartTimeApplications.length).toBe(2);

      const [firstSFASPartTimeApplication, secondSFASPartTimeApplication] =
        sfasPartTimeApplications;

      expect(firstSFASPartTimeApplication).toEqual({
        startDate: startDate,
        endDate: endDate,
        CSGPAward: 4000,
        SBSDAward: 300,
        CSPTAward: 30000,
        CSGDAward: 165,
        BCAGAward: 24,
        CSLPAward: 500,
        programYearId: 20232024,
      });

      expect(secondSFASPartTimeApplication).toEqual({
        startDate: startDate,
        endDate: endDate,
        CSGPAward: 5000,
        SBSDAward: 400,
        CSPTAward: 40000,
        CSGDAward: 166,
        BCAGAward: 25,
        CSLPAward: 600,
        programYearId: 20232024,
      });
    },
  );
```